playbooks/besman-scorecard-0.0.1-playbook.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+function __besman_init() {
+    __besman_echo_white "initialising"
+    export ASSESSMENT_TOOL_NAME="scorecard"
+    export ASSESSMENT_TOOL_TYPE="scorecard"
+    export ASSESSMENT_TOOL_VERSION="v4.13.1"
+    export ASSESSMENT_TOOL_PLAYBOOK="besman-$ASSESSMENT_TOOL_TYPE-0.0.1-playbook.sh"
+    
+    local steps_file_name="besman-$ASSESSMENT_TOOL_NAME-0.0.1-steps.sh"
+    export BESMAN_STEPS_FILE_PATH="$BESMAN_PLAYBOOK_DIR/$steps_file_name"
+
+    local var_array=("BESMAN_ARTIFACT_TYPE" "BESMAN_ARTIFACT_NAME" "BESMAN_ARTIFACT_VERSION" "BESMAN_ARTIFACT_URL" "BESMAN_ENV_NAME" "BESMAN_ARTIFACT_DIR" "ASSESSMENT_TOOL_NAME" "ASSESSMENT_TOOL_TYPE" "ASSESSMENT_TOOL_VERSION" "ASSESSMENT_TOOL_PLAYBOOK" "BESMAN_ASSESSMENT_DATASTORE_DIR" "BESMAN_ASSESSMENT_DATASTORE_URL" "BESMAN_LAB_OWNER_TYPE" "BESMAN_LAB_OWNER_NAME")
```

Comment:
Please change the variables - BESLAB_OWNER_TYPE -> BESMAN_LAB_TYPE; BESLAB_OWNER_NAME -> BESMAN_LAB_NAME